app/javascript/components/map/map-decode-config.js

```diff
@@ -110,33 +110,33 @@ export default {
     }
   },
   // GLADs
-  '95b34906-d998-496b-80e7-9d8e2a70cd25': {
+  'dd5df87f-39c2-4aeb-a462-3ef969b20b66': {
```

Comment:
This is begging to be put in a constant somewhere

Reply:
This is the constants. We can have a separate one for mapping to the layer Id, but then we have to cross compare again inside the layer selector.

Reply:
nah it's fine. Didn't realised that...